### PR TITLE
Fix Bug 1151279 - Create itisatrap.org/firefox/its-unwanted.html

### PR DESCRIPTION
**Pull request by fmarier**
This is a test page for the new "potentially unwanted software"
list from Google:

  http://googleonlinesecurity.blogspot.com.au/2015/03/even-more-unwanted-software-protection.html

### CODE REVIEW

**Comment by fmarier** [comment]
@bensternthal who do you think I should get to review the new copy?

**Comment by bensternthal** [comment]
I would say Matej for copy. Does this need to go through a legal review? My default is always to have legal look and decide and would suggest that here.

File: css/its-a-trap.css [hunk 1]
```diff
@@ -11,17 +13,21 @@
     width: 430px;
 }
 
+* html #its-unwanted #main-feature,
 * html #its-a-trap #main-feature,
 * html #its-an-attack #main-feature { height: 265px; /* Hack for IE6 */ }
```
Comment (bensternthal) [inline css/its-a-trap.css]: Should there be IE6 fixes in here?

My understanding is that this page is invoked from inside the browser.

File: firefox/its-unwanted.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    </div><!-- end #wrapper -->
+
+    <!-- start #footer -->
+    <div id="footer">
+    <div id="footer-contents">
+    <div id="copyright">
+        <p id="footer-links"><a href="http://www.mozilla.org/en-US/privacy-policy.html">Privacy Policy</a> &nbsp;|&nbsp;
+        <a href="http://www.mozilla.org/en-US/about/legal.html">Legal Notices</a> &nbsp;|&nbsp;
+                    <a href="http://www.mozilla.org/en-US/legal/fraud-report/index.html">Report Trademark Abuse</a></p>
+        <p>Except where otherwise <a href="/en-US/about/legal.html#site">noted</a>, content on this site is licensed under the <br /><a href="http://creativecommons.org/licenses/by-sa/3.0/">Creative Commons Attribution Share-Alike License v3.0</a> or any later version.</p>
```
Comment (bensternthal) [inline firefox/its-unwanted.html]: Broken link.

**Comment by bensternthal** [comment]
Outside of the PR comments, the design on these pages is looking dated. I'm not sure if we want to address this here but we may want to look at updating these pages to better match what mozilla properties look like. Thoughts?

**Comment by fmarier** [comment]
> Outside of the PR comments, the design on these pages is looking dated. I'm not sure if we want to 
> address this here but we may want to look at updating these pages to better match what mozilla 
> properties look like. Thoughts?

I think you're right, these pages are quite old and the issues you found above (both of which are in the existing boilerplate and affect all three pages) are a good sign of that.

I've just filed issue #5 for that since it's a separate piece of work from just adding another test page.

File: css/its-a-trap.css (hunk 1 above)
Comment (fmarier) [inline css/its-a-trap.css]: > My understanding is that this page is invoked from inside the browser.

I'm not sure what you mean by "invoked inside the browser" but this is a page that users might be taken to from SUMO or other documentation in order to test that they have Safe Browsing turned on.

**Comment by fmarier** [comment]
@bensternthal I've made the copy changes requested by Matej in https://bugzilla.mozilla.org/show_bug.cgi?id=1151279#c5 and also fixed the broken link you found in all three pages.

I'll mention this when I ask legal before landing the new list support in Firefox, but for now I think it's OK to put the page up (I will use it to test my patch) since we're not going to link to it from anywhere until the feature lands.

**Comment by bensternthal** [comment]
Tested locally and R+

To get this deployed to production you can file a bug under the following product and component, just mention the URL of this repository. I will cc you on an old bug so you have an example.

Infrastructure & Operations -> WebOps:Other

**Pull request merged**